basichtml.js

```diff
@@ -2,6 +2,7 @@ const utils = require('./src/utils');
 const CustomElementRegistry = require('./src/CustomElementRegistry');
 const Document = require('./src/Document');
 const HTMLElement = require('./src/HTMLElement');
+const HTMLUnknownElement = require('./src/HTMLElement');
```

Comment:
this makes no sense. It should be `class extends HTMLElement {}`, not exactly the same class. Yet I don't see any valid use case for that in here.

Reply:
@WebReflection fixed the import you were right. What do you mean, you don't see a use case in here?

Reply:
I mean: what are you trying to solve ? why do you need `HTMLUnknownElement` ? what is the use case ?

Reply:
See #28 for discussion 👍